git/task-set-commit-status.yaml

```diff
@@ -282,6 +287,9 @@ spec:
         target_url = ("https://cloud.ibm.com/devops/pipelines/tekton/" + os.environ["PIPELINE_ID"] +
           "/runs/" + os.environ["PIPELINE_RUN_ID"] + "?env_id=ibm:yp:" + previous_step["TOOLCHAIN_REGION"])
 
+        if "$(params.target-url)" != "":
```

Comment:
I would rather use an `if then else` to compute the target_url or only use it from the given parameter.
With this current code change, the default target_url is always computed even if there is a provided one

Reply:
@jauninb I have updated the PR.